src/components/drivers/pdf-viewer.jsx

```diff
@@ -123,6 +123,17 @@ export default class PDFDriver extends React.Component {
     }
   }
 
+  componentWillUnmount() {
+    // Only utilize PDFJS.getDocument() if isEvalSupported == false.
+    if (PDFJS.isEvalSupported == false) {
```

Comment:
Any reason you're not using strict equality here of `===`?

Reply:
old C# habits. Fix it.